ui/packages/catalog-ui-search/src/main/webapp/component/location-old/location-old.view.js

```diff
@@ -106,9 +106,17 @@ module.exports = Marionette.LayoutView.extend({
   updateMaxAndMin: function() {
     this.model.setLatLon()
   },
+  isFilterUndefinedOrNull: function(filter) {
+    if (filter === null || filter.value == undefined || filter.value == null) {
```

Comment:
Should we be using the identity comparison for filter.value instead of the equivalence comparison?

Reply:
It should actually be `if (filter === null || filter.value === undefined || filter.value === null) {`
I could also do  `if (!filter && !filter.value && !filter.value) {`
but I don't know which one is more readable, I'm guessing the former.